event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinNviReport.java

```diff
@@ -14,7 +14,8 @@ public record CristinNviReport(String publicationIdentifier,
                                List<CristinLocale> cristinLocales,
                                String yearReported,
                                PublicationDate publicationDate,
-                               String instanceType) implements JsonSerializable {
+                               String instanceType,
+                               Object reference) implements JsonSerializable {
```

Comment:
Why not create a `Reference` class?

Reply:
I need publicationInstance and publicationContext from this one. Plan is to extract using jsonNode, as we do in nvi when creating UpsertCandidateRequest. No need for class named Reference containing to other objects.